docs/deployment/deploy-without-docker/Load-Sample-Cancer-Study.md

```diff
@@ -4,7 +4,19 @@ Once you have confirmed that the cBioPortal server is installed,
 you are ready to import data. Importing a sample study is recommended
 to verify that everything is working correctly.
 
-The cBioPortal distribution includes a [small dummy study, `study_es_0`](https://github.com/cBioPortal/cbioportal/tree/master/core/src/test/scripts/test_data/study_es_0), which contains all datatypes supported by cBioPortal. This document describes how to import the prerequisites for the sample study and how to import the study itself.
+cBioPortal Core has a [small dummy study, `study_es_0`](https://https://github.com/cBioPortal/cbioportal-core/tree/main/src/test/scripts/test_data/study_es_0), which contains all datatypes supported by cBioPortal. This document describes how to import the prerequisites for the sample study and how to import the study itself.
+
+
+## Download and Build cBioPortal Core
+
+```
+	git clone https://github.com/cBioPortal/cbioportal-core.git
+	cd cbioportal-core
+	git checkout main
+	mvn -DskipTests clean install
+```
+
+
 
 ## Set the PORTAL_HOME environment variable
```

Comment:
is this now only used by cbioportal-core?

Reply:
Yep, we got rid of a lot of dependencies